src/ring_theory/adjoin_root.lean

```diff
@@ -31,6 +32,7 @@ The main definitions are in the `adjoin_root` namespace.
 
```

Comment:
Should some of the new results from this PR be added to the module docstring?

Reply:
I added a bit to the docstring

Reply:
Looks like your additions disappeared in the merge.

Reply:
Those results aren't in adjoin_root anymore, they are in power_basis.

Reply:
Should I add them to the power_basis docstring?

Reply:
Oh, I see. Yeah, that would be a good idea. I think also `lift_hom` and `equiv` could potentially be mentioned in this module docstring, unless you think that they wouldn't be used much outside this file.